SonarRunner.Shim/SonarRunner.Wrapper.cs

```diff
@@ -127,11 +127,13 @@ private static string FindRunnerExe(AnalysisConfig config, ILogger logger)
             return success;
         }
 
-        private static void WarnAgainstSettingSonarRunnerHome(ILogger logger)
+        private static void IgnoreSonarRunnerHome(ILogger logger)
         {
-            if (!String.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable(SonarRunnerHomeVariableName)))
+            if (!String.IsNullOrWhiteSpace(
+                Environment.GetEnvironmentVariable(SonarRunnerHomeVariableName)))
             {
                 logger.LogWarning(Resources.WARN_SonarRunnerHomeIsSet);
```

Comment:
@bgavrilMS If we need a message at all for this, then `LogMessage()` seems more appropriate IMO